lib/remote_translations/caller.rb

```diff
@@ -40,7 +42,7 @@ def translations
 
     def fields_values
       resource.translated_attribute_names.map do |field|
-        resource.send(field)
+        sanitize(resource.send(field))
```

Comment:
In order to keep the behaviour we had before commit 7bf4e4d6, I think we need to use `WYSIWYGSanitizer.new.sanitize` :thinking:. Then we can remove the line saying `include ActionView::Helpers::SanitizeHelper`. Correct me if I'm wrong! :wink: